### PR TITLE
Update toc.yml

### PR DESCRIPTION
**Pull request by willtebbutt**
Follow up from https://github.com/TuringLang/TuringTutorials/pull/422

### CODE REVIEW

File: _data/toc.yml [hunk 1]
```diff
@@ -85,8 +85,8 @@
   - title: "Probabilistic PCA"
     url: "tutorials/11-probabilistic-pca"
 
-  - title: "Gaussian Processes"
-    url: "tutorials/12-gaussian-process"
+  - title: "GPLVM"
```
Comment (torfjelde) [inline _data/toc.yml:88]: I'd prefer the full name here, or at last "Gaussian Process LVM" or something.

Comment (willtebbutt) [reply _data/toc.yml:88]:
```suggestion
  - title: "Gaussian Process LVM"
```

**Review by torfjelde — APPROVED** [review]
LGTM!

**Comment by willtebbutt** [comment]
Doesn't look like I have merge rights. If you wouldn't mind hitting the button @torfjelde

**Pull request merged**